docs/reference/research/puck_physics_spec.md

```
@@ -0,0 +1,774 @@
+FROM ref_moneypuck_shots
+GROUP BY angle_band
+ORDER BY angle_band;
+```
+Expected: highest xG at 0-10 degrees (directly in front), drops toward 90 degrees.
+
+**Test 5: Board COR validation**
+If your CV pipeline can detect rimmed pucks frame-by-frame, measure:
+```
+COR_measured = |v_out_normal| / |v_in_normal|
+```
+Compare to REBOUND_COR["board_hdpe_frozen"]["cor"] = 0.40.
```

markdownlint-cli2 (0.21.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: ""cor""

(MD052, reference-links-images)